packages/app/src/components/time-series-chart/components/timespan-annotation.tsx

```diff
@@ -127,3 +89,29 @@ export function TimespanAnnotationIcon({
     </svg>
   );
 }
+
+export function HatchedTimespanAnnotationIcon() {
+  return (
+    <svg height="15" width="15">
```

Comment:
Can you make sure this icon has the same width/height/viewbox implementation as the other icon?
```suggestion
export function HatchedTimespanAnnotationIcon({
  width = 15,
  height = 15,
}: TimespanAnnotationIconProps) {
  return (
    <svg width={width} height={height} viewBox={`0 0 ${width} ${height}`}>
```

Reply:
Done